homeassistant/components/template/config_flow.py

```diff
@@ -104,19 +113,14 @@ def generate_schema(domain: str, flow_type: str) -> dict[vol.Marker, Any]:
 def options_schema(domain: str) -> vol.Schema:
     """Generate options schema."""
     return vol.Schema(
-        {vol.Required(CONF_STATE): selector.TemplateSelector()}
-        | generate_schema(domain, "option"),
+        generate_schema(domain, "option"),
```

Comment:
`generate_schema` should return the `vol.Schema` object directly to avoid duplicated lines. In addition, we can use `partial` to create `options_schema` and `config_schema` functions